.changeset/light-insects-bow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sap-cloud-sdk/generator": minor
+---
+
+[Fix] Filter for operations without OperationImport. Unnecessay warning logs are no longer generated.
```

Comment:
[req] It is the other way around, I think. The operation imports don't have an operation. Please note, that changelogs should be written in imperative and implementation details are not needed.

```suggestion
[Fix] Fix incorrect warning logs for operation imports that do have a matching operation in the service specification.
```